Procurement/ViewModel/Recipes/RareSetRecipe.cs

```diff
@@ -5,12 +5,20 @@
 
 namespace Procurement.ViewModel.Recipes
 {
+    public enum SetType
+    {
+        Normal,
+        Shaper,
+        Elder
```

Comment:
As in the other PR, I'd recommend adding a trailing comma here, but, again, it's a minor style nitpick.